app/models/question_set.rb

```diff
@@ -26,4 +16,32 @@ def find(id)
 
     Question.new(@questions.fetch(id))
   end
+
+  class QuestionSetLoader
+    def initialize(partial_checker:)
+      @partial_checker = partial_checker
+      @root_url = Rails.root
```

Comment:
`@root_url` doesn't appear to be used anywhere..? As an aside, `Rails.root` is the absolute path to the root of the app rather than a URL.

Reply:
Oops! that was left over from the refactoring. 

Thanks for the aside! 👍 

Have amended this commit